.github/workflows/bump_version_release.yaml

```diff
@@ -0,0 +1,75 @@
+name: Bump version and release
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        type: choice
+        required: true
+        options:
+          - "RF24"
+          - "RF24Network"
+          - "RF24Mesh"
+          - "RF24Gateway"
+          - "RF24Ethernet"
```

Comment:
These are the repos that this new workflow can be used on.

Reply:
I added RF24Audio to this list just in case we ever decide to start issuing new releases for that lib.

Reply:
Although this was specially designed for our C++ libs, I think this would also work for our python libs as well. To do so, I would just need to add the python project/repo name to this list.

Maybe some time in the future... if I'm feeling special kind of lazy 🤣